plugin/pkg/scheduler/algorithmprovider/defaults/defaults.go

```diff
@@ -95,6 +95,18 @@ func init() {
 	factory.RegisterFitPredicate("MatchNodeSelector", predicates.PodSelectorMatches)
 	// Optional, cluster-autoscaler friendly priority function - give used nodes higher priority.
 	factory.RegisterPriorityFunction("MostRequestedPriority", priorities.MostRequestedPriority, 1)
+	// Cluster autoscaler friendly scheduling algorigthm.
+	factory.RegisterAlgorithmProvider("ClusterAutoscalerProvider", defaultPredicates(),
```

Comment:
nit: can you please register it next the the other provider (at the beginning of this function)?